Duplicati/Library/Backend/Jottacloud/Jottacloud.cs

```diff
@@ -49,6 +50,21 @@ public class Jottacloud : IBackend, IStreamingBackend
         private readonly System.Net.NetworkCredential m_userInfo;
         private readonly byte[] m_copybuffer = new byte[Duplicati.Library.Utility.Utility.DEFAULT_BUFFER_SIZE];
 
+        private static readonly string JFS_DEFAULT_CHUNKSIZE = "5mb";
+        private static readonly string JFS_DEFAULT_THREADS = "4";
+        private readonly int m_threads;
+        private readonly int m_chunksize;
```

Comment:
It looks like this value is always being compared to `longs`.  Is there a reason why we declare it to be an `int` here?

Reply:
Agreed